.history/src/content/index_20260124163513.js

```
@@ -0,0 +1,213 @@
+chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
+    logger.info('收到消息:', request);
+    
+    switch (request.action) {
+        case 'toggleClipboardMonitoring':
+            toggleClipboardMonitoring();
+            sendResponse({ success: true });
+            break;
+            
+        case 'getSelectedText':
+            const selectedText = window.getSelection().toString().trim();
+            sendResponse({ selectedText });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)